Makefile

```diff
@@ -141,3 +146,19 @@ update-dependencies:
 .PHONY: add-license-headers
 add-license-headers: $(GO_ADD_LICENSE)
 	@./hack/addlicenseheaders.sh ${YEAR}
+	
+.PHONY: kind-up
+kind-up: kind-down
```

Comment:
We should not delete the cluster before creating it when running `make kind-up`. If the cluster already exists, a warning should be raised so that the caller can decide whether to delete it or not.

Reply:
I think he addressed my comment: https://github.com/gardener/etcd-druid/pull/547#issuecomment-1481219169 but I agree with @seshachalam-yv now better to check whether kind cluster exist with same name if it does exist then throw warning.

Reply:
I think, it's okay to not delete the cluster before starting. If the user gets error due to already existing cluster, they can see the error and take action on basis of that. No need to check whether the cluster exist explicitly..

Reply:
Addressed